docs/source/user_data/create-datasets-for-dashboard.ipynb

```diff
@@ -8,18 +8,31 @@
     "\n",
     "The following notebook steps through how to add a dataset to the MAAP Dashboard.\n",
     "\n",
-    "The MAAP dashboard has 3 environments:\n",
+    "Note, there are 2 scenarios:\n",
+    "\n",
+    "1. Adding a single Cloud-Optimized GeoTIFF (COG) and,\n",
+    "2. Adding multiple spatially distinct COGs as a \"mosaic\" with mosaicJSON.\n",
```

Comment:
they don't actually need to be spatially distinct, overlap is handled via a parameter. But point taken that it results in a single mosaic not a multi-dimensional (e.g. time series).